doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -4,120 +4,72 @@
+* Use exsiting specifications and detailed well documented Test Plan / Test Cases
+* Add addioinal test cases where needed
+* Integrate with Upstream Projects and OVP process (code, docs, cert criteria, etc.)
+* Have clear pass/fail criteria
 
-<a name="1.5"></a>
-## 1.5 Best Practices
-
-The following best practices have been adopted to ensure verification and validation procedures are repeatable with consistent quality in test results, and RI conformances:
-
-* Standardized test methodology / flow, Test Plan, and Test Case Suites
-* Integration with OPNFV Upstream Projects and OVP flow (code, docs, cert criteria, etc.)
-* Leverage Network and Service Models, with identified VNF-specific parameters
-* Standardized conformance criteria
-* Define CNTT RA as scenarios, and have all test cases for the RA be involved in OVP (could also be addressed in OVP as CNTT test)
-* Add test cases from operators, which operators already tested in their environment
-
-<a name="1.6"></a>
-## 1.6 Verification methodologies
+<a name="1.4"></a>
+## 1.4 Performance Testing
 
-Perform VNF interoperability verifications against an implementation of CNTT reference architecture, leveraging existing OPNFV Intake Process. Upstream projects will define features/capabilities, test scenarios, and test cases to augment existing OVP test harnesses to be executed via the OVP Ecosystem.
+Prior to onboarding a VNF the infrastructure must be tested in the Network, Compute and Storage domains to ensure that it provides sufficient resources and capabilities for the VNF to behave as expected (i.e. deliver capacity and performance as forecast). Infrastructure vendors are expected to go through extensive testing and optimization at component and stack level prior to testing for **conformance**.
 
-3rd Party test platforms may also be leveraged, if desired.
+Performance measurements may be extremely sensitive to hardware and software configurations and test methods. Testing should therefore be as realistic as possible i.e. to ensure a test environment that represents a realistic Cloud Infrastructure setup using open-source tools and test VNFs (i.e. workloads) that mimic as much as possible a real VNF consuming available infrastrcuture resources. Test methods that directly measure performance from the perspective of the VNF are more realistic than infering VNF performance by testing "under-the-hood" attributes of the infrastructure. Testing the infrastructre from the viewpoint of the VNF ultimately helps to ensure that behaviour of commercial VNFs will be more predictable and if a commercial VNF does not perform well, the tools, methods and reference results from RC may be very useful for investigating the discrepancy. Furterhmore, test tools that are "stand-alone" and packaged as VNFs can be deployed using the same onboarding methods as a commercial VNFs and hence will be portable across differnt commerical infrastructures.
```

Comment:
The RA's specify the hardware and software configurations. CNTT expects that the vendors conform to these as otherwise we will have, similar to what exists today, multiple different configurations to contend with